src/common/libraries/MessageLib.sol

```diff
@@ -108,21 +106,16 @@ library MessageLib {
         (89  << uint8(MessageType.RevokedShares) * 8) +
         (89  << uint8(MessageType.DepositRequest) * 8) +
         (89  << uint8(MessageType.RedeemRequest) * 8) +
-        (105 << uint8(MessageType.FulfilledDepositRequest) * 8) +
-        (105 << uint8(MessageType.FulfilledRedeemRequest) * 8) +
+        (121 << uint8(MessageType.FulfilledDepositRequest) * 8) +
+        (121 << uint8(MessageType.FulfilledRedeemRequest) * 8) +
         (73  << uint8(MessageType.CancelDepositRequest) * 8) +
         (73  << uint8(MessageType.CancelRedeemRequest) * 8) +
-        (89  << uint8(MessageType.FulfilledCancelDepositRequest) * 8) +
-        (89  << uint8(MessageType.FulfilledCancelRedeemRequest) * 8) +
         (82  << uint8(MessageType.UpdateHoldingAmount) * 8) +
         (50  << uint8(MessageType.UpdateShares) * 8) +
         (73  << uint8(MessageType.TriggerIssueShares) * 8) +
-        (25  << uint8(MessageType.TriggerSubmitQueuedShares) * 8);
-
-    // forgefmt: disable-next-item
-    uint256 constant MESSAGE_LENGTHS_2 =
-        (41 << (uint8(MessageType.TriggerSubmitQueuedAssets) - 32) * 8) +
-        (26 << (uint8(MessageType.SetQueue) - 32) * 8);
+        (25  << uint8(MessageType.TriggerSubmitQueuedShares) * 8) +
+        (41 << uint8(MessageType.TriggerSubmitQueuedAssets) * 8) +
+        (26 << uint8(MessageType.SetQueue) * 8);
```

Comment:
Leaving a breadcrumb here for when we exceed 32 messages again in the future

Reply:
https://github.com/centrifuge/protocol-v3/pull/345/files will make it go to #33 btw